internal/config/config.go

```diff
@@ -62,13 +74,13 @@ type Secret struct {
 	SecretArgs map[string]interface{} `yaml:"secretArgs,omitempty"`
 }
 
-func Parse(parametersStr, targetPath, permissionStr string) (Config, error) {
+func Parse(logger hclog.Logger, parametersStr, targetPath, permissionStr string) (Config, error) {
```

Comment:
Wondering if we really need the logger being passed in on a parse function. I see you're logging a warning but do we think we'll need that information?

Reply:
Yeah it felt a bit weird. I just wanted to try to do something to surface the deprecated configs for people who miss the release notes. I think I'll remove this after one release if that sounds ok to you.

Reply:
Sounds good to me.